### ISSUE TITLE
Inconsistent electronic states for [CH] methylidyne

### ISSUE THREAD

**Issue by rwest** [ReactionMechanismGenerator/RMG-database#172]
Very much like #171 for [C], I think the libraries have a variety of representations for [CH] which may not have been checked carefully.  For example, why does GRI-Mech have a doublet CH(D) in the transport library but quartet CH in the kinetics and thermo libraries?

According to Wikipedia:
  The ground state is a doublet radical with one unpaired electron,
  and the first two excited states are a quartet radical with three
  unpaired electrons and a doublet radical with one unpaired electron.
  With the quartet radical only 71 kJ above the ground state, a sample
  of methylidyne exists as a mixture of electronic states even at
  room temperature, giving rise to complex reactions.

So perhaps we should default to making [CH] the doublet radical with one unpaired electron?
(though how it reacts in families, I cannot say...? might that accidentally make our models worse..? would you get there from [CH2]?) 

It might be that we want them all (including GRI transport) to be quartets, as most of them currently are. Anyway, this issue is just to make sure someone thinks about it and checks these are as intended.

```
./kinetics/libraries/combustion_core/version2/dictionary.txt-CH
./kinetics/libraries/combustion_core/version2/dictionary.txt-multiplicity 4
./kinetics/libraries/combustion_core/version2/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/combustion_core/version2/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/combustion_core/version3/dictionary.txt-CH
./kinetics/libraries/combustion_core/version3/dictionary.txt-multiplicity 4
./kinetics/libraries/combustion_core/version3/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/combustion_core/version3/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/combustion_core/version4/dictionary.txt-CH
./kinetics/libraries/combustion_core/version4/dictionary.txt-multiplicity 4
./kinetics/libraries/combustion_core/version4/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/combustion_core/version4/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Dooley/C1/dictionary.txt-CH
./kinetics/libraries/Dooley/C1/dictionary.txt-multiplicity 4
./kinetics/libraries/Dooley/C1/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Dooley/C1/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Dooley/methylformate/dictionary.txt-CH
./kinetics/libraries/Dooley/methylformate/dictionary.txt-multiplicity 4
./kinetics/libraries/Dooley/methylformate/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Dooley/methylformate/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-CH
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-multiplicity 4
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-CH
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-multiplicity 4
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt-CH
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt-multiplicity 4
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/FFCM1(-)/dictionary.txt-CH
./kinetics/libraries/FFCM1(-)/dictionary.txt-multiplicity 4
./kinetics/libraries/FFCM1(-)/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/FFCM1(-)/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/GRI-Mech3.0/dictionary.txt-CH
./kinetics/libraries/GRI-Mech3.0/dictionary.txt-multiplicity 4
./kinetics/libraries/GRI-Mech3.0/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/GRI-Mech3.0/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt-CH
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt-multiplicity 4
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/JetSurF0.2/dictionary.txt-CH
./kinetics/libraries/JetSurF0.2/dictionary.txt-multiplicity 4
./kinetics/libraries/JetSurF0.2/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/JetSurF0.2/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt-CH
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt-multiplicity 4
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt-CH
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt-multiplicity 4
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt-CH
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt-multiplicity 4
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt:1 C u3 p0 c0 {2,S}
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt-2 H u0 p0 c0 {1,S}
--
--
./thermo/libraries/Chernov.py-    label = "CH",
./thermo/libraries/Chernov.py-    molecule = 
./thermo/libraries/Chernov.py-"""
./thermo/libraries/Chernov.py-multiplicity 4
./thermo/libraries/Chernov.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/Chernov.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/Chernov.py-""",
--
--
./thermo/libraries/DFT_QCI_thermo.py-    label = "CH",
./thermo/libraries/DFT_QCI_thermo.py-    molecule = 
./thermo/libraries/DFT_QCI_thermo.py-"""
./thermo/libraries/DFT_QCI_thermo.py-multiplicity 4
./thermo/libraries/DFT_QCI_thermo.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/DFT_QCI_thermo.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/DFT_QCI_thermo.py-""",
--
--
./thermo/libraries/FFCM1(-).py-    label = "CH",
./thermo/libraries/FFCM1(-).py-    molecule = 
./thermo/libraries/FFCM1(-).py-"""
./thermo/libraries/FFCM1(-).py-multiplicity 4
./thermo/libraries/FFCM1(-).py:1 C u3 p0 c0 {2,S}
./thermo/libraries/FFCM1(-).py-2 H u0 p0 c0 {1,S}
./thermo/libraries/FFCM1(-).py-""",
--
--
./thermo/libraries/GRI-Mech3.0-N.py-    label = "CH",
./thermo/libraries/GRI-Mech3.0-N.py-    molecule = 
./thermo/libraries/GRI-Mech3.0-N.py-"""
./thermo/libraries/GRI-Mech3.0-N.py-multiplicity 4
./thermo/libraries/GRI-Mech3.0-N.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/GRI-Mech3.0-N.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/GRI-Mech3.0-N.py-""",
--
--
./thermo/libraries/GRI-Mech3.0.py-    label = "CH",
./thermo/libraries/GRI-Mech3.0.py-    molecule = 
./thermo/libraries/GRI-Mech3.0.py-"""
./thermo/libraries/GRI-Mech3.0.py-multiplicity 4
./thermo/libraries/GRI-Mech3.0.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/GRI-Mech3.0.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/GRI-Mech3.0.py-""",
--
--
./thermo/libraries/JetSurF0.2.py-    label = "CH",
./thermo/libraries/JetSurF0.2.py-    molecule = 
./thermo/libraries/JetSurF0.2.py-"""
./thermo/libraries/JetSurF0.2.py-multiplicity 4
./thermo/libraries/JetSurF0.2.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/JetSurF0.2.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/JetSurF0.2.py-""",
--
--
./thermo/libraries/Narayanaswamy.py-    label = "CH(14)",
./thermo/libraries/Narayanaswamy.py-    molecule = 
./thermo/libraries/Narayanaswamy.py-"""
./thermo/libraries/Narayanaswamy.py-multiplicity 4
./thermo/libraries/Narayanaswamy.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/Narayanaswamy.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/Narayanaswamy.py-""",
--
--
./thermo/libraries/USC-Mech-ii.py-    label = "CH",
./thermo/libraries/USC-Mech-ii.py-    molecule = 
./thermo/libraries/USC-Mech-ii.py-"""
./thermo/libraries/USC-Mech-ii.py-multiplicity 4
./thermo/libraries/USC-Mech-ii.py:1 C u3 p0 c0 {2,S}
./thermo/libraries/USC-Mech-ii.py-2 H u0 p0 c0 {1,S}
./thermo/libraries/USC-Mech-ii.py-""",
--
--
./transport/libraries/GRI-Mech.py-    label = "CH(D)",
./transport/libraries/GRI-Mech.py-    molecule = 
./transport/libraries/GRI-Mech.py-"""
./transport/libraries/GRI-Mech.py-multiplicity 2
./transport/libraries/GRI-Mech.py:1 C u1 p1 c0 {2,S}
./transport/libraries/GRI-Mech.py-2 H u0 p0 c0 {1,S}
./transport/libraries/GRI-Mech.py-""",
```

**Comment by alongd** [comment]
https://github.com/ReactionMechanismGenerator/RMG-database/pull/180 addresses most of the instances here, except for ERC and combustion_core.

**Comment by KEHANG** [comment]
ERC and combustion_core now are also addressed by #180.
